### PR TITLE
EOS-17113: SSPL dev deploy script failed to create repos for specific build(-T)

### PR DESCRIPTION
**Pull request by mariyappanp**

### CODE REVIEW

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy [hunk 1]
```diff
@@ -180,7 +174,7 @@ class SSPLDevDeploy:
                print(f"ERROR: Unable to download {pkg_name}.")
             cmd = "yum install -y %s%s" % (build_url, pkg[0])
             returncode = self._execute_cmd(cmd)
-            if not (returncode==0 or returncode==256):
+            if returncode != 0:
```
Comment (satish-seagate) [inline low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy:177]: Is it fine to remove returncode==256 here?
I am seeing that you have added this earlier.

**Review by satish-seagate — APPROVED** [review]

**Review by shriya-deshmukh — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy (hunk 1 above)
Comment (mariyappanp) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy:177]: Yes, if pkg is already installed, we had two different cases.
Now we uninstall cort-py-utils from cleanup(). So we expect returncode 0 for this fresh installation.

**Review by satish-seagate — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy [hunk 2]
```diff
@@ -169,7 +163,7 @@ class SSPLDevDeploy:
             _ = self._execute_cmd(cmd)
 
             # cortx-py-utils dependency
-            cmd = "python3 -m pip -r https://raw.githubusercontent.com/Seagate/cortx-utils/main/py-utils/requirements.txt"
+            cmd = "python3 -m pip install -r https://raw.githubusercontent.com/Seagate/cortx-utils/main/py-utils/requirements.txt"
```
Comment (sandeepsgt) [inline low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy:166]: so cortx-py-utils is not installing its dependencies ?

Comment (sandeepsgt) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy:166]:
```Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Installing : cortx-py-utils-1.0.0-102_23eb199.noarch                                                                              1/1
WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3.6 install --user` instead.
Collecting aiohttp==3.6.1 (from -r /opt/seagate/cortx/utils/conf/requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/e0/cb/5fa1d7722488995358d6092a7d2ccceb9011ab78842ccd336fbb5ec0d878/aiohttp-3.6.1-cp36-cp36m-manylinux1_x86_64.whl```

I observed that it gets installed with `cortx-py-utils`.

**Review by cdeshmukh — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy (hunk 2 above)
Comment (mariyappanp) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy:166]: It wasn't earlier. Now we can see its dependencies also getting installed.
I will remove this line.

Comment (mariyappanp) [reply low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy]: Done.

**Review by sandeepsgt — APPROVED** [review]

**Pull request merged**